File: programming_ref/python/exampleClass.py
class testClass:

	def __init__(self):
		self.var1 = 5
		self.var2 = "test class"

	def testFunction(self, addN):
		self.var1 += addN

	def printState(self):
		print("var1 is ", self.var1)
		print("var2 is ", self.var2)

File: programming_ref/cpp/exampleClass.cpp
#include <iostream>
#include <string>
#include "exampleClass.hpp"

exampleClass::exampleClass() {
    std::cout << "Class has been created" << std::endl;
}

void exampleClass::setInt(int x) {
    var1 = x;
}

void exampleClass::setStr(std::string x) {
    var2 = x;
}

void exampleClass::printState() {
    std::cout << "var 1 is " << var1 << std::endl;
    std::cout << "var 2 is " << var2 << std::endl;
}
